### PR TITLE
CI via Github Actions

### LINKED ISSUE #15: set up circleci to run tests on each PR

### PR DESCRIPTION
**Pull request by danielhollas**
Let's try continuous integrations via Github Actions.

Closes #15

### CODE REVIEW

**Comment by mpolyak** [comment]
@danielhollas it's saying there's a conflict in `tests/assistant_spec.js`, is that something you can resolve so I may squash & merge?

**Comment by danielhollas** [comment]
@mpolyak I rebased, it should be fine now. However, for some reason the CI is no longer triggered on this PR. Do you have any idea why?

I also tried to resolve the test flakiness, see the second commit.

File: tests/assistant_spec.js [hunk 1]
```diff
@@ -16,50 +16,45 @@ const {MATH_RULES_LOCALES} = require('../lib/math-translator');
  */
 function makeGraphie() {
     const baseURL = 'web+graphie://ka-perseus-graphie.s3.amazonaws.com';
-    const id = Date.now() + (1000 * Math.random() | 0);
+    const id = Date.now() + (100000 * Math.random() | 0);
+    console.log(id);
```
Comment (mpolyak) [inline tests/assistant_spec.js:20]: Can this console.log be removed?

File: tests/assistant_spec.js [hunk 2]
```diff
@@ -1148,145 +1142,177 @@ describe('TranslationAssistant (graphie)', function() {
...
-        assertSuggestions(allItems, itemsToTranslate, [graphie2]);
+        assertSuggestions(allItems, itemsToTranslate, [graphies[1]]);
     });
 });
 
 describe('TranslationAssistant (image and graphie links)', function() {
+    // NOTE(danielhollas): Testing whether some test flakiness
+    // is caused by non-unique graphie or image links
+    it('tests should have unique Graphies', function() {
+        for (let i = 0; i < 6; i++) {
```
Comment (mpolyak) [inline tests/assistant_spec.js:1195]: Can you make `6` a file `const` please or use `graphies.length`?

Comment (danielhollas) [reply tests/assistant_spec.js:1196]: Yep, done.

File: tests/assistant_spec.js (hunk 1 above)
Comment (danielhollas) [reply tests/assistant_spec.js]: Huh, I was too hasty. It's weird that ESLint didn't catch it. Apparently, the new version does not have the `no-console` rule by default, so I added it to `.eslintrc`. Thanks!

**Comment by danielhollas** [comment]
@mpolyak thanks, should be ready now. I thought that the Github Action is automatically picked up when the specified file is added, but apparently not. :-)

File: tests/assistant_spec.js (hunk 2 above)
Comment (mpolyak) [reply tests/assistant_spec.js:1196]: I still see `for (let i = 0; i < 6; i++)`?

Comment (danielhollas) [reply tests/assistant_spec.js]: I'm sorry, I fixed it on one place but not here. Should be okay now.

**Comment by mpolyak** [comment]
@danielhollas looks like workflow succeeded but seems the badge in README is not setup correctly for: https://github.com/Khan/translation-assistant/actions?query=workflow%3A%22Node.js+CI%22 ?

**Comment by mpolyak** [comment]
ps. If if the README.md is going to be corrected maybe mention `.git-blame-ignore-revs`?

Something like:
```
To ignore eslint style-guide commits in blame history, use:
git blame --ignore-revs-file .git-blame-ignore-revs

or to do it automatically:
git config blame.ignoreRevsFile .git-blame-ignore-revs
```

**Comment by danielhollas** [comment]
@mpolyak thanks! I rebased the other PRs and the workflow seems to work. :tada:  I'll fix the README badge as part of one of them.

**Pull request merged**